Lib/idlelib/idle_test/test_textview.py

```diff
@@ -96,5 +96,52 @@ def test_view_file(self):
         self.assertIsNone(view)
 
 
+class ButtonClickTextViewTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.root = Tk()
+        cls.root.withdraw()
+        cls._utest = True
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.root.update_idletasks()
+        cls.root.destroy()
+        del cls.root
+
+    def setUp(self):
+        self.view = None
+        self.called = False
+
+    def tearDown(self):
+        if self.view:
+            self.view.destroy()
+
+    def test_view_text_bind_with_button(self):
+        def _command():
+            self.called = True
+            self.view = tv.view_text(self.root, 'TITLE_TEXT', 'COMMAND', _utest=self._utest)
+        button = Button(self.root, text='BUTTON', command=_command)
+        button.invoke()
+
+        self.assertEqual(self.called, True)
+        self.assertEqual(self.view.title(), 'TITLE_TEXT')
+        self.assertEqual(self.view.textView.get('1.0', '1.end'), 'COMMAND')
+
+    def test_view_file_bind_with_button(self):
+        def _command():
+            fn = findfile('CREDITS.txt', 'idlelib')
```

Comment:
osp = os.path
fn =osp.abspath(osp.join(osp.dirname('__file__'), '..', 'CREDITS.txt'))
would be equivalent.  But to make the test simpler, just pass '__file__' instead of fn and adjust assert.